src/DataFetcher.py

```diff
@@ -8,6 +9,9 @@
 from src.YahooFinance import YahooFinanceQuoteSummary, YahooFinanceQuoteSummaryModule
 from threading import Lock
 
+logger = logging.getLogger("IsThisStockGood")
```

Comment:
Can we revert this file (except for the new Logger lines: 1, 12, 240, 242, 244?) to eliminate all the other whitespace changes?

(it may be simpler to revert to HEAD and just re-add those few lines back)